### PR TITLE
Feature/#275 cv monochrome force create studies section

### PR DESCRIPTION
**Pull request by deletidev**

### CODE REVIEW

File: packages/manfred2html/src/engine/cv-monochrome-force/html-parts/experience-section/experience-section.ejs [hunk 1]
```diff
@@ -17,7 +17,7 @@
         </div>
         <%_}-%>
         <%_ if(experience.url && experience.url !== "undefined") { -%>
```
Comment (juanpms2) [inline packages/manfred2html/src/engine/cv-monochrome-force/html-parts/experience-section/experience-section.ejs:19]: remove && experience.url !== "undefined", please

File: packages/manfred2html/src/engine/cv-monochrome-force/html-parts/experience-section/experience-section.ejs [hunk 2]
```diff
@@ -17,7 +17,7 @@
...
-        <a href="https://www.getmanfred.com" class="experience__url"><%- experience.url %></a>
+        <a href="https://www.getmanfred.com" class="experience__url" target="_blank"><%- experience.url %></a>
```
Comment (juanpms2) [inline packages/manfred2html/src/engine/cv-monochrome-force/html-parts/experience-section/experience-section.ejs:20]: use experience.url in href

File: packages/manfred2html/src/engine/cv-monochrome-force/html-parts/studies-section/studies-section.ejs [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    <%_ for (const study of studiesCollection) { -%>
+        <div class="education">
+          <div class="decoration">
+            <div class="decoration__circle"></div>
+            <hr class="decoration__line" />
+          </div>
+          <div class="education__group">
+            <p class="education__date"><%- new Date(study.startDate).toLocaleDateString(undefined, { year:"numeric", month:"short"})%> <%_ if(study.finishDate && study.finishDate.toLowerCase() !=='actualidad') { -%>
+              <%- "- "+new Date(study.finishDate).toLocaleDateString(undefined, { year:"numeric", month:"short"}) -%>
+            <%_ } else { -%> - Actualidad
```
Comment (juanpms2) [inline packages/manfred2html/src/engine/cv-monochrome-force/html-parts/studies-section/studies-section.ejs:13]: Create new label: CURRENT_DATE: 'string'

spanish -> CURRENT_DATE: 'Actualidad'
english -> CURRENT_DATE: 'now'

File: packages/manfred2html/src/engine/cv-monochrome-force/html-parts/experience-section/experience-section.ejs (hunk 1 above)
Comment (deletidev) [reply packages/manfred2html/src/engine/cv-monochrome-force/html-parts/experience-section/experience-section.ejs]: It's fixed

File: packages/manfred2html/src/engine/cv-monochrome-force/html-parts/experience-section/experience-section.ejs (hunk 2 above)
Comment (deletidev) [reply packages/manfred2html/src/engine/cv-monochrome-force/html-parts/experience-section/experience-section.ejs]: It's fixed

File: packages/manfred2html/src/engine/cv-monochrome-force/html-parts/studies-section/studies-section.ejs (hunk 3 above)
Comment (deletidev) [reply packages/manfred2html/src/engine/cv-monochrome-force/html-parts/studies-section/studies-section.ejs]: It's done and it's also done in the experience section.

**Pull request merged**